src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,10 @@ Do you want to proceed?</value>
   <data name="TheRenameTrackingSessionWasCancelledAndIsNoLongerAvailable" xml:space="preserve">
     <value>The rename tracking session was cancelled and is no longer available.</value>
   </data>
+  <data name="Rename2" xml:space="preserve">
+    <value>Rename '{0}'</value>
+  </data>
+  <data name="SingleQuote" xml:space="preserve">
```

Comment:
If another language wants to do something roughly equivalent to this:
```
'identifier' is being renamed
```

Then the only way they could do that with this scheme is to localize ```SingleQuote``` to ``` ' is being renamed ```